equinix/resource_fabric_connection_acc_test.go

```diff
@@ -0,0 +1,244 @@
+		}
+		order {
+		purchase_order_number = "1-129105284100"
+			}
+		bandwidth = %d
+		a_side {
+		access_point {
+			type = "COLO"
+				port {
+				uuid = "c4d9350e-77cd-7cdd-1ce0-306a5c00a600"
+				}
+				link_protocol {
+					type= "QINQ"
+					vlan_s_tag= 1231
+				}
+			}
+		}
+		z_side {
+		access_point {
+			type = "COLO"
+				port{
+				uuid = "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+				}
+				link_protocol {
+					type= "QINQ"
+					vlan_s_tag= 457
+				}
+			location {
+        		metro_code= "SV"
+      		}
+			}
+		}
+	}`, bandwidth)
+}
+
+func testAccFabricCreateAzureConnectionConfig(bandwidth int32) string {
+	return fmt.Sprintf(`resource "equinix_fabric_connection" "test" {
+	name = "fabric_tf_acc_CSAZURE"
+	description = "Test Connection"
+	type = "EVPL_VC"
+	notifications{
+		type="ALL" 
+		emails=["example@equinix.com"]
+	} 
+	bandwidth = %d
+	redundancy {priority= "PRIMARY"}
+	order {
+    	purchase_order_number= "1-323292"
+  	}
+  	a_side {
+    	access_point {
+      		type= "COLO"
+      		port {
+        		uuid= "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+      		}
+      	link_protocol {
+        	type= "QINQ"
+        	vlan_s_tag= "2019"
+      	}
+    }
+  }
+  	z_side {
+    	access_point {
+      		type= "SP"
+			authentication_key= "a38565b9-5d32-45ba-bb01-0649e2735753"
+      		profile {
+        		type= "L2_PROFILE"
+        		uuid= "bfb74121-7e2c-4f74-99b3-69cdafb03b41"
+      		}
+      		location {
+        		metro_code= "SV"
+      		}
+    	}
+  	}
+}
+`, bandwidth)
+}
+
+func testAccFabricUpdateConnectionConfig(bandwidth int32) string {
+	return fmt.Sprintf(`resource "equinix_fabric_connection" "test" {
+	uuid = equinix_fabric_connection.test.uuid
+	name = "fabric_tf_acc_CSAZURE"
+	description = "Test Connection"
+	type = "EVPL_VC"
+	notifications{
+		type="ALL" 
+		emails=["example@equinix.com"]
+	} 
+	bandwidth = %d
+	redundancy {priority= "PRIMARY"}
+	order {
+    	purchase_order_number= "1-323292"
+  	}
+  	a_side {
+    	access_point {
+      		type= "COLO"
+      		port {
+        		uuid= "c4d9350e-783c-83cd-1ce0-306a5c00a600"
+      		}
+      	link_protocol {
+        	type= "QINQ"
+        	vlan_s_tag= "2019"
+      	}
+    }
+  }
+  	z_side {
+    	access_point {
+      		type= "SP"
+			authentication_key= "a38565b9-5d32-45ba-bb01-0649e2735753"
+      		profile {
+        		type= "L2_PROFILE"
+        		uuid= "bfb74121-7e2c-4f74-99b3-69cdafb03b41"
+      		}
+      		location {
+        		metro_code= "SV"
+      		}
+    	}
+  	}
+}
+`, bandwidth)
+}
+
+func TestAccFabricReadConnection(t *testing.T) {
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccFabricReadConnectionConfig(),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"equinix_fabric_connection.test", "name", fmt.Sprint("fabric_tf_acc_test")),
```

Comment:
I can't test it but this test doesn't look good to me, there's is just a data source in the config 
[testAccFabricReadConnectionConfig](https://github.com/syedluri/terraform-provider-equinix/blob/c331f164209c90e21b051a6c32b79413cf5abc16/equinix/resource_fabric_connection_acc_test.go#L240) so test needs to check that data source "**data.**equinix_fabric_connection.test", same below with ImportState, that works for resources

Reply:
This is known issues, thanks for catching. Before going on PTO, this was failing but as it was only test case I did not remove it. But now I fixed, I missed the "data" part of the resource name.